Makefile

```diff
@@ -16,25 +17,25 @@ package: check-version
 	# Copy files to pristine temporary directory
 	rm -rf $(TEMP_PATH)	
 	mkdir $(TEMP_PATH)
+	rm -rf ${TEMP_PATH_DEBIAN}/usr/lib/tcltk/tcl8/site-tcl
+	mkdir -p ${TEMP_PATH_DEBIAN}/usr/lib/tcltk/tcl8/site-tcl
+	rm -rf ${TEMP_PATH_DEBIAN}/DEBIAN
+	mkdir -p ${TEMP_PATH_DEBIAN}/DEBIAN
 	rm -rf package
 	mkdir package
 	curl --fail -K ~/.curlrc_github -L -o v$(VERSION).tar.gz https://api.github.com/repos/qcode-software/qcode-tcl/tarball/v$(VERSION)
 	tar --strip-components=1 --exclude Makefile --exclude description-pak --exclude doc --exclude docs.tcl --exclude package.tcl \
 	-xzvf v$(VERSION).tar.gz -C $(TEMP_PATH)
-	./package.tcl $(TEMP_PATH)/tcl package ${NAME} ${VERSION}
-	./pkg_mkIndex package ${NAME} ${VERSION}
-	# checkinstall
-	fakeroot checkinstall -D --deldoc --backup=no --install=no --pkgname=$(DPKG_NAME) --pkgversion=$(VERSION) --pkgrelease=$(RELEASE) \
-	-A all -y --maintainer $(MAINTAINER) --pkglicense="BSD" --reset-uids=yes --requires "tcl,tcllib,html2text,curl,tclcurl" \
-	--replaces none --conflicts none make install
+	./package.tcl $(TEMP_PATH)/tcl ${TEMP_PATH_DEBIAN}/usr/lib/tcltk/tcl8/site-tcl ${NAME} ${VERSION}
+	./control.tcl ${TEMP_PATH_DEBIAN}/DEBIAN ${DPKG_NAME} ${VERSION} ${RELEASE} ${MAINTAINER}
+	# build package
+	dpkg-deb --build ${TEMP_PATH_DEBIAN} $(DPKG_NAME)_$(VERSION)-$(RELEASE)_all.deb
```

Comment:
Think you need ```fakeroot``` here otherwise the dev's file ownership will leak into the debian package.
eg.
```
$ dpkg -c qcode-tcl-14.2.0_14.2.0-0_all.deb
drwxr-xr-x david/david       0 2024-10-07 16:03 ./
drwxr-xr-x david/david       0 2024-10-07 16:03 ./usr/
drwxr-xr-x david/david       0 2024-10-07 16:03 ./usr/lib/
drwxr-xr-x david/david       0 2024-10-07 16:03 ./usr/lib/tcltk/
drwxr-xr-x david/david       0 2024-10-07 16:03 ./usr/lib/tcltk/tcl8/
drwxr-xr-x david/david       0 2024-10-07 16:04 ./usr/lib/tcltk/tcl8/site-tcl/
-rw-r--r-- david/david  775677 2024-10-07 16:04 ./usr/lib/tcltk/tcl8/site-tcl/qcode-14.2.0.tm
```
vs
```
$ dpkg -c qcode-tcl-14.2.0_14.2.0-0_all.deb
drwxr-xr-x root/root         0 2024-10-07 16:11 ./
drwxr-xr-x root/root         0 2024-10-07 16:03 ./usr/
drwxr-xr-x root/root         0 2024-10-07 16:03 ./usr/lib/
drwxr-xr-x root/root         0 2024-10-07 16:03 ./usr/lib/tcltk/
drwxr-xr-x root/root         0 2024-10-07 16:11 ./usr/lib/tcltk/tcl8/
drwxr-xr-x root/root         0 2024-10-07 16:11 ./usr/lib/tcltk/tcl8/site-tcl/
-rw-r--r-- root/root    775677 2024-10-07 16:11 ./usr/lib/tcltk/tcl8/site-tcl/qcode-14.2.0.tm
```

Reply:
Updated